modules/administration/pages/troubleshooting/tshoot-hostname-rename.adoc

```diff
@@ -47,25 +47,37 @@ The only mandatory parameter for the script is the newly configured IP address o
 . Reboot the {productname} Server to use the new network configuration and to ensure the hostname has changed.
 . Run the script [command]``spacewalk-hostname-rename`` script with the public IP address of the server.
     If the server is not using the new hostname, the script fails.
-    Be aware that this script refreshes the pillar data for all Salt minion: the time it takes to run depends on the number of registered Salt systems.
-. Re-configure your clients to make your environment aware of the new hostname and IP address.
-    In the Salt minion configuration file [path]``/etc/salt/minion``, you must specify the name of the new Salt master ({productname} Server):
+    Be aware that this script refreshes the pillar data for all {salt} clients: the time it takes to run depends on the number of registered clients.
+
+. *Skip this step if the clients are managed via a {productname} proxy.*
+  Re-configure the clients directly managed to make them aware of the new hostname and IP address.
+  In the {salt} client configuration file, you must specify the name of the new {salt} master ({productname} Server) (the filename is [path]``/etc/venv-salt-bundle/minion``or, if you do not use the {salt} bundle, [path]``/etc/salt/minion``):
 +
 ----
 master: <new_hostname>
 ----
-+
-. Once this is changed, restart the [literal]``salt-minion` process:
+
+. Restart the [literal]``salt-minion`` process:
 +
 ----
 systemctl restart salt-minion
 ----
```

Comment:
I think this also needs to be adjusted for the Salt bundle (`vens-salt-minion`) vs. Salt (`salt-minion`).

Reply:
thanks for your patience!